CHANGES.md

```diff
@@ -12,6 +12,8 @@
 
 + core: Add `AsyncComponentStream` to supports streams for async components as well
 + core: Builder pattern for initializing factories similar to regular components
++ core: Implement `RelmContainerExt` for libadwaita 1.4's `NavigationView`
++ core: Implement `RelmSetChildExt` for libadwaita 1.4's `NavigationPage`
```

Comment:
And I just noticed, this has to moved to `## Unreleased` because we can't add the current changes to an already released version of Relm4. So please add a `### Added` category underneath `## Unreleased` :)

Reply:
Ah, my bad! I'll quickly fix those things.

Reply:
I hope this does it. If there are any other issues let me know! ^^

And don't worry about the response time. I'm aware that you are busy and that is completely fine.